.github/pull_request_template.md

```diff
@@ -23,3 +23,11 @@ Please update the following to link the associated issue. This is required for s
 -->
 
 Fixes: #issue_number
+
+## Contributor checklist
+Please verify that the PR meets the following requirements, where applicable:
+
+- [ ] An overview of proposed schema changes is included in a linked GitHub issue.
+- [ ] A design document PR is created in the [design-notes repository](https://github.com/radius-project/design-notes/), if new APIs are being introduced.
+- [ ] A PR for the [samples repository](https://github.com/radius-project/samples) is created, if existing samples are affected by the changes in this PR.
```

Comment:
do we want to include more possibilities so we , say, 'add new samples when new features are introduced'. 
One suggestion:
..ensure samples repo adequately addresses changes in this PR (which may include addition/update/deletion) of samples

Reply:
That should be covered in the the design review. This list captures what we agreed on in the team meeting.